.zenodo.json

```diff
@@ -1,114 +1,118 @@
-    },
-    {
-      "affiliation": "UC Berkeley - UCSF Graduate Program in Bioengineering",
-      "name": "Keshavan, Anisha",
-      "orcid": "0000-0003-3554-043X"
-    },
-    {
-      "affiliation": "Dartmouth College, Hanover, NH, United States",
-      "name": "Ma, Feilong"
-    },
-    { 
-      "name": "Christian, Horea"
-    },
-    { 
-      "name": "Poelen, Jorrit"
-    },
-    { 
-      "name": "Skytén, Kusti"
-    },
-    {
-      "affiliation": "UC Berkeley",
-      "name": "Visconti di Oleggio Castello, Matteo",
-      "orcid": "0000-0001-7931-5272"
-    },
-    { 
-      "name": "Hardcastle, Nell"
-    },
-    { 
-      "name": "Stoeter, Torsten"
-    },
-    { 
-      "name": "C Lau, Vicky"
-    },
-    {
-      "affiliation": "Stanford University, Stanford, CA, United States",
-      "name": "Markiewicz, Christopher J.",
-      "orcid": "0000-0002-6533-164X"
-    },
-    {
-      "affiliation": "Psychoinformatics Lab, INM-7, Research Centre Juelich",
-      "name": "Wagner, Adina S.",
-      "orcid": "0000-0003-2917-3450"
-    },
-    {
-      "affiliation": "Maze Therapeutics, South San Francisco, CA, United States",
-      "name": "Nichols, B. Nolan",
-      "orcid": "0000-0003-1099-3328"
-    }
-  ],
-  "grants": [
-    {"id": "10.13039/100000001::1429999"}
-  ],
-  "keywords": [
-    "data management",
-    "data distribution",
-    "execution provenance tracking",
-    "version control"
-  ],
-  "access_right": "open",
-  "license": "MIT",
-  "upload_type": "software"
-}
+    "creators": [
+        {
+            "affiliation": "Institute of Neuroscience and Medicine, Brain & Behaviour (INM-7), Research Centre J\u00fclich, J\u00fclich, Germany and Institute of Systems Neuroscience, Medical Faculty, Heinrich Heine University D\u00fcsseldorf, D\u00fcsseldorf, Germany",
+            "name": "Hanke, Michael",
+            "orcid": "0000-0001-6398-6370"
+        },
+        {
+            "affiliation": "Dartmouth College, Hanover, NH, United States",
+            "name": "Halchenko, Yaroslav O.",
+            "orcid": "0000-0003-3456-2493"
+        },
+        {
+            "affiliation": "Institute of Neuroscience and Medicine, Brain & Behaviour (INM-7), Research Centre J\u00fclich, J\u00fclich, Germany",
+            "name": "Poldrack, Benjamin",
+            "orcid": "0000-0001-7628-0801"
+        },
+        {
+            "affiliation": "Dartmouth College, Hanover, NH, United States",
+            "name": "Meyer, Kyle"
+        },
+        {
+            "affiliation": "Dartmouth College, Hanover, NH, United States",
+            "name": "Solanky, Debanjum Singh"
+        },
+        {
+            "name": "Alteva, Gergana"
+        },
+        {
+            "affiliation": "Dartmouth College, Hanover, NH, United States",
+            "name": "Gors, Jason"
+        },
+        {
+            "name": "MacFarlane, Dave"
+        },
+        {
+            "name": "Olaf H\u00e4usler, Christian",
```

Comment:
unicode `ä` is lost.

Reply:
This is likely a bug with saving json, so for the meantime I'll change it back to be what is currently there.

Reply:
This is fixed with the library now.